packages/evm/src/precompiles/index.ts

```diff
@@ -1,6 +1,12 @@
-import { Address, isTruthy } from '@ethereumjs/util'
 import { Common, Hardfork } from '@ethereumjs/common'
-import { PrecompileInput, PrecompileFunc } from './types'
+import { Address, isTruthy } from '@ethereumjs/util'
+
+import { precompile0a } from './0a-bls12-g1add'
+import { precompile0b } from './0b-bls12-g1mul'
+import { precompile0c } from './0c-bls12-g1multiexp'
+import { precompile0d } from './0d-bls12-g2add'
+import { precompile0e } from './0e-bls12-g2mul'
+import { precompile0f } from './0f-bls12-g2multiexp'
```

Comment:
In this particular case I think those 6 should still go after the { precompile09 } line